src/js/components/visualization/ChartComponent.jsx

```diff
@@ -83,15 +85,13 @@ export class ChartComponent extends Component {
         data={chartData}
         drilldown={drillDownFieldName != null}
         options={options}
-        ref='chart'
+        ref={(ref) => (this.chart = ref)}
```

Comment:
is there a reason to not just just `ref='chart'` and then reference it as `this.refs.chart`? is this another weird ref issue with react?